docs/blog/2019-01-04-seo-and-gatsby/index.md

```diff
@@ -0,0 +1,449 @@
+---
+title: "Search Engine Optimization with Gatsby"
+date: 2019-01-04
+author: Dustin Schau
+cover: images/seo.jpg
+excerpt: "SEO and Gatsby: A Perfect Pairing. Learn how Gatsby implements SEO utilizing React Helmet and smart defaults and how you can use these tools to implement your own!"
```

Comment:
I've been relentlessly trained to prefer "using" to "utilizing" simply because it's one syllable shorter.

Reply:
It could also be shorter "Learn how to use Gatsby to implement SEO with React Helmet and smart defaults!